packages/bulk-edit/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@theia/bulk-edit",
+  "version": "1.7.0",
+  "description": "Theia - Bulk Edit Extension",
+  "dependencies": {
+    "@theia/core": "^1.7.0",
+    "@theia/editor": "^1.7.0",
+    "@theia/filesystem": "^1.7.0",
+    "@theia/monaco": "^1.7.0",
+    "@theia/workspace": "^1.7.0"
```

Comment:
Please be sure to also update the following dependencies to `^1.8.0`.
The build currently does not succeed due to it (verified locally).

Reply:
Sorry about that. must have not saved file before pushing.  Fixed